lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -190,6 +191,14 @@ data TransactionLayer k tx = TransactionLayer
         --
         -- Returns `Nothing` for ByronEra transactions.
 
+    , estimateSignedTransactionSize
+        :: Node.ProtocolParameters
+        -> tx
+        -> Maybe TxSize
+        -- ^ Estimate the size of the transaction when fully signed.
+        --
+        -- Returns `Nothing` for ByronEra transactions.
```

Comment:
Would be neat to abstract out this pattern of returning `Nothing` for ByronEra transactions, but I'm intentionally sticking with it for now.